tests/functional/create_subnets_test.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2018 Cable Television Laboratories, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from drb_python.http_exceptions import AuthorizationError, ConnectionError
+from drb_python.drb_exceptions import ActionError, AlreadyExists
+from drb_python.subnet import Subnet
+from drb_python.http_session import HttpSession
+
+
+class SubnetTest(unittest.TestCase):
+    """
+    Tests for functions located in SubnetHttps
+    """
+    def test_create_subnet(self):
```

Comment:
Please create tearDown() method to cleanup the Subnet being created to ensure it will happen nearly all of the time. By having the delete within the test method, an exception could cause this to fail.

Reply:
Also done